paasta_tools/deployd/master.py

```diff
@@ -259,6 +259,7 @@ def prioritise_bouncing_services(self):
 
     def start_watchers(self):
         """ should block until all threads happy"""
+        watchers_enabled = self.config.get_watchers_enabled()
```

Comment:
Can you rename this to `disabled_watchers`?

It is much more likely we need to toggle off the watchers we don't want, I don't want people to have to enumerate all the watchers they want, then subtract.

Plus it makes the logic less weird on line 276